README.md

```diff
@@ -6,7 +6,7 @@
 
 PyScript is a framework that allows users to create rich Python applications in the browser using HTML's interface and the power of [Pyodide](https://pyodide.org/en/stable/), [MicroPython](https://micropython.org/) and [WASM](https://webassembly.org/), and modern web technologies.
 
-To get started see the [getting started tutorial](docs/tutorials/getting-started.md).
+To get started see [Beginning PyScript](https://github.com/pyscript/docs/blob/main/docs/beginning-pyscript.md).
```

Comment:
```suggestion
To get started see [Beginning PyScript](https://docs.pyscript.net/latest/beginning-pyscript/).
```
Changed to the docs page directly with a "prettier" link that'll always grab the latest version.

Thanks for finding this issue!